src/Nethermind/Nethermind.Evm/CodeAnalysis/IL/ILCompiler.cs

```diff
@@ -1260,6 +1260,85 @@ private static void EmitSegmentBody(Emit<ExecuteSegment> method, OpcodeInfo[] co
                     method.LoadLocalAddress(localSpan);
                     method.Call(typeof(Span<byte>).GetMethod(nameof(Span<byte>.CopyTo), [typeof(Span<byte>)]));
                     break;
+                case Instruction.LOG0:
+                case Instruction.LOG1:
+                case Instruction.LOG2:
+                case Instruction.LOG3:
+                case Instruction.LOG4:
+                    method.StackLoadPrevious(stack, head, 1);
+                    method.Call(Word.GetUInt256);
+                    method.StoreLocal(uint256A);
+                    method.StackLoadPrevious(stack, head, 2);
+                    method.Call(Word.GetUInt256);
+                    method.StoreLocal(uint256B);
+                    method.StackPop(head, 2);
+
+                   //topics count
+                    var topicsCount = method.DeclareLocal<long>();
+                    method.LoadConstant((long)op.Operation);
+                    method.LoadConstant((long)Instruction.LOG0);
+                    method.Subtract();
+                    method.StoreLocal(topicsCount);
```

Comment:
I am not able to find a method in Sigil that allows me to create a constant. I could write c# instead of IL  `const topicsCount = Op.Operation - Instruction.Log0`  or incorporate `using` from the previous comment ?

Reply:
you can do ``method.LoadConstant(Op.Operation - Instruction.Log0);`` ?